addon/components/item-picker/component.js

```diff
@@ -39,6 +39,9 @@ export default Component.extend({
   init () {
     this._super(...arguments);
     this.set('itemsToAdd', []);
+    if (this.get('priorSelections')) {
+      this.set('itemsToAdd', this.get('priorSelections'));
+    }
```

Comment:
I think I would rather just allow passing in `itemsToAdd`. So you would replace lines 41 - 44 with:

```
if (!this.get('itemsToAdd')) {
  this.set('itemsToAdd', []);
}
```